From f570679582d3de6b502bc9c24312937714636ae2 Mon Sep 17 00:00:00 2001
From: paolo <paolo@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Wed, 25 May 2011 09:45:58 +0000
Subject: [PATCH] 2011-05-24  Paolo Carlini  <paolo.carlini@oracle.com>

	PR libstdc++/49141
	* testsuite/26_numerics/complex/cons/48760.cc: Use dg-require-c-std.
	* testsuite/26_numerics/headers/cmath/19322.cc: Likewise.



git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_6-branch@174179 138bc75d-0d04-0410-961f-82ee72b054a4

index 0201cc7..0027a34 100644
--- a/libstdc++-v3/testsuite/26_numerics/complex/cons/48760.cc
+++ b/libstdc++-v3/testsuite/26_numerics/complex/cons/48760.cc
@@ -1,3 +1,5 @@
+// { dg-require-c-std "" }
+
 // Copyright (C) 2011 Free Software Foundation, Inc.
 //
 // This file is part of the GNU ISO C++ Library.  This library is free
diff --git a/libstdc++-v3/testsuite/26_numerics/headers/cmath/19322.cc b/libstdc++-v3/testsuite/26_numerics/headers/cmath/19322.cc
index 955286f..25494c3 100644
--- a/libstdc++-v3/testsuite/26_numerics/headers/cmath/19322.cc
+++ b/libstdc++-v3/testsuite/26_numerics/headers/cmath/19322.cc
@@ -1,4 +1,6 @@
-// Copyright (C) 2005, 2009 Free Software Foundation, Inc.
+// { dg-require-c-std "" }
+
+// Copyright (C) 2005, 2009, 2010, 2011 Free Software Foundation, Inc.
 //
 // This file is part of the GNU ISO C++ Library.  This library is free
 // software; you can redistribute it and/or modify it under the
@@ -15,11 +17,9 @@
 // with this library; see the file COPYING3.  If not see
 // <http://www.gnu.org/licenses/>.
 
-
 #include <cmath>
 #include <testsuite_hooks.h>
 
-#if _GLIBCXX_USE_C99_MATH
 // libstdc++/19322
 void test01()
 {
@@ -27,12 +27,9 @@ void test01()
 
   VERIFY( !std::isnan(3.0) );
 }
-#endif
 
 int main()
 {
-#if _GLIBCXX_USE_C99_MATH
   test01();
-#endif
   return 0;
 }
-- 
1.7.0.4

